src/main/java/thePackmaster/cards/energyandechopack/Slowing.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.cards.energyandechopack;
+
+import com.megacrit.cardcrawl.cards.status.Wound;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.BerserkPower;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.powers.energyandechopack.ReceptorPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.applyToSelf;
+import static thePackmaster.util.Wiz.makeInHand;
+
+public class Slowing extends AbstractPackmasterCard {
+
+    public final static String ID = makeID(Slowing.class.getSimpleName());
+
+    private static final int COST = 2;
+
+    public Slowing(){
+        super(ID, COST, CardType.POWER, CardRarity.UNCOMMON, CardTarget.SELF);
+        magicNumber = baseMagicNumber = 3;
+        cardsToPreview = new Wound();
+    }
+
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        applyToSelf(new BerserkPower(p, 1));
```

Comment:
Do you want to add your own power for this instead of reusing Berserk?

Reusing Berserk does work for giving energy, but it will make the [E] icons show up as Ironclad energy icons (a bit weird) and the player will see a card named Slowing give a buff named Berserk (also weird).

Making a buff called Extra Energy or something like that, with some icon for it (maybe use the Packmaster's energy orb, or adapt the icon used by Charged Battery), would make the buff a lot more fitting.